downloads/platform.md

```diff
@@ -76,9 +76,16 @@ You can uninstall Julia by deleting Julia.app and the packages directory in `~/.
 
 ## Linux and FreeBSD
 
-It is strongly recommended that the official generic binaries from the downloads page be used to install Julia on Linux and FreeBSD.
+It is strongly recommended that the official generic binaries from the downloads page be used to install Julia on Linux and FreeBSD. The following
+command downloads the latest Julia version and makes a symlink to allow the `julia` command to find the binary:
 
+```
+wget https://julialang-s3.julialang.org/bin/linux/x64/1.4/julia-1.4.2-linux-x86_64.tar.gz
+tar -xvzf julia-1.4.2-linux-x86\_64.tar.gz
+ln -s "$PWD"/julia-1.4.2/bin/julia ~/.local/bin/
```

Comment:
Is this in `PATH` by default?

Reply:
It's kind of weird that these instructions are just run in some unspecified directory. Presumably the user's home directory? Maybe a little info about where to run this?

Reply:
is there a place we should download it to?

Reply:
Not really, but maybe a word about "pick a place where you want the Julia directory to live—this can be your home directory or maybe a `~/dev` directory if you prefer that.

Reply:
When you `apt-get` or install a Windows binary, it just goes somewhere and that's where it is. What's a good default for us to do here? Anyone who cares where it goes will know what to do: the instructions are for the other audience.

Reply:
Why not say that run the command from the directory you want it in?